articles/libraries/_includes/_configure_custom_domain.md

```diff
@@ -0,0 +1,29 @@
+### Configure your custom domain
+
+1. Ensure that your application updates are made prior to setting up custom domains in your application, as the older versions of ${library} will not work when configured with a custom domain.
+
+2. Set up your [custom domain](/custom-domains) in the Auth0 [Dashboard](${manage_url}/#/tenant).
+
+3. Use the custom domain when instantiating Lock.
+
+```
+var lock = new Auth0Lock('your-client-id', 'login.northwind.com', options);
```

Comment:
That's the first time we see `login.northwind.com` in the article, and it made me stop and think for a moment. Maybe replace it with sth like `login.your-domain.com`?
Also use client ID var.